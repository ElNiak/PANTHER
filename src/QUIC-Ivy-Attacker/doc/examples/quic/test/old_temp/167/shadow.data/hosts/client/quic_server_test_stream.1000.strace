00:00:00.000000000 [tid 1000] shadow_init_memory_manager(...) = 0
000000000000000000 [tid 1000] gettimeofday(...) = 0
00:00:00.000000000 [tid 1000] openat(-100, "/sys/devices/system/cpu", O_RDONLY | O_NONBLOCK | O_DIRECTORY | O_CLOEXEC, (empty)) = 3
00:00:00.000001010 [tid 1000] fstat(...) = 0
00:00:00.000001010 [tid 1000] getdents64(...) = 752
00:00:00.000003010 [tid 1000] getdents64(...) = 0
00:00:00.000003010 [tid 1000] close(3) = 0
00:00:00.000005010 [tid 1000] sched_getaffinity(499, 8, 0x555555e1eac0) = -3 (ESRCH)
000000000000005010 [tid 1000] clock_gettime(...) = 0
00:00:00.000006020 [tid 1000] openat(-100, "/sys/devices/system/cpu/online", O_RDONLY | O_CLOEXEC, (empty)) = 3
00:00:00.000006020 [tid 1000] read(3, 0x7fffffffc1e0, 8192) = 2
00:00:00.000008020 [tid 1000] close(3) = 0
00:00:00.000008020 [tid 1000] futex(...) = 0
00:00:00.000010020 [tid 1000] futex(...) = 0
00:00:00.000010020 [tid 1000] setrlimit(...) = <native>
000000000000012020 [tid 1000] ^^^ = 0
000000000000012020 [tid 1000] time(...) = 946684800
00:00:00.000012020 [tid 1000] write(2, "\nARG__ \n", 8) = 8
00:00:00.000013030 [tid 1000] write(2, "0xb", 3) = 3
00:00:00.000013030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000015030 [tid 1000] write(2, "770", 3) = 3
00:00:00.000015030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000017030 [tid 1000] write(2, "\nARG__ \n", 8) = 8
00:00:00.000017030 [tid 1000] write(2, "0xd", 3) = 3
00:00:00.000019030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000019030 [tid 1000] write(2, "772", 3) = 3
00:00:00.000021030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000021030 [tid 1000] write(2, "\nARG__ \n", 8) = 8
00:00:00.000023030 [tid 1000] write(2, "1", 1) = 1
00:00:00.000023030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000025030 [tid 1000] write(2, "1", 1) = 1
00:00:00.000025030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000027030 [tid 1000] write(2, "\nARG__ \n", 8) = 8
00:00:00.000027030 [tid 1000] write(2, "1", 1) = 1
00:00:00.000029030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000029030 [tid 1000] write(2, "1", 1) = 1
00:00:00.000031030 [tid 1000] write(2, "\n", 1) = 1
00:00:00.000031030 [tid 1000] futex(...) = 0
00:00:00.000033030 [tid 1000] write(2, "syntax error in parameter value iversion"..., 41) = 41
00:00:00.000033030 [tid 1000] exit_group(...) = <native>
