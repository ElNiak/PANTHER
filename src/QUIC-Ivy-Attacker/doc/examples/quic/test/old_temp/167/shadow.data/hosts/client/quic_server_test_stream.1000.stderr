
ARG__ 
0xb
770

ARG__ 
0xd
772

ARG__ 
1
1

ARG__ 
1
1
syntax error in parameter value iversion
